tasks/README.md

```diff
@@ -84,6 +84,7 @@ The preferred style of the keys should be `snake_case`.
 
 **remote**: Default `false`. All implementation of this task operate on a remote target using the `_target` metaparam.(rev 4)
 
+**tags**: An array of strings used for arbitrary tags without inherent, semantic meaning to Bolt. Useful for categorizing tasks.
```

Comment:
We shouldn't mention Bolt here. This spec is for any task runner.

Reply:
Fixed